### PR TITLE
Extract postsecondary matriculation counts

### PR DESCRIPTION
**Pull request by chiouey**
## proposed changes
- draft `enrich_grad_count()` to calculate and return matriculation counts
  - accounts for 16 mo vs. 12 mo matriculation rates
- changes in linked branches override

## test
`testthat::test_file("tests/testthat/test_report_card.R")`

### CODE REVIEW

File: R/grate.R [hunk 1]
```diff
@@ -903,3 +903,87 @@ gcount_column_order <- function(df) {
...
+  )
+  
+  grad_counts <- grad_counts %>%
+    bind_rows() %>%
+    select(-c(county_name, district_name, school_name,
+              cohort_count))
+    
+  postsec_rates <- data.frame(
+    end_year = 2012:2019,
+    is_16mo = c(T, T, T, T, T, F, F, F)
```
Comment (almartin82) [inline R/grate.R:943]: Love it!

File: R/grate.R [hunk 2]
```diff
@@ -903,3 +903,87 @@ gcount_column_order <- function(df) {
...
+    mutate(gc_year = if_else(is_16mo, end_year - 1, as.numeric(end_year))) %>%
+    mutate(
+      subgroup = tolower(subgroup),
+      subgroup = case_when(
+        subgroup == "economically disadvantaged students" ~ "economically disadvantaged",
+        subgroup %in% c("english language learners",
+                        "english learners") ~ "limited english proficiency",
+        subgroup == "students with disabilities" ~ "students with disability",
+        subgroup == "two or more races" ~ "multiracial",
+        subgroup == "native hawaiian" ~ "pacific islander", # or native amer.?
```
Comment (almartin82) [inline R/grate.R:960]: these are the special population subgroup names, correct?  I like consistency across these PR files - that's good.  Maybe we should make a note about weirdness in how subgroups have been collapsed by DOE across various years and we could print it to the console as a `warning()`?  if that sounds good let's open an issue for that as an enhancement (🚢  🚢  🚢 )

**Comment by chiouey** [comment]
> I think this is all but there! I see two ground truth tests that were failing for me -
> 
> ```
> test_report_card.R:130: failure: extract_rc_college_matric ground truth values
> `%>%`(...) not equal to 85.3.
> 1/1 mismatches
> [1] 67.2 - 85.3 == -18.1
> 
> test_report_card.R:137: failure: extract_rc_college_matric ground truth values
> `%>%`(...) not equal to 93.6.
> 1/1 mismatches
> [1] 87.4 - 93.6 == -6.2
> ```
> 
> Might be a 12 month vs 16 month thing? Can we check these before merging?

fixed! i must have been looking at the wrong flat file while writing the test and then didn't bother checking it 😴

**Comment by almartin82** [comment]
Looks good!

There was a merge conflict on `geo.R` - I took a look at both files and resolved in 3179c34 - @chiouey can you take a look at that and confirm? `ward_gcount_aggs` goes away and we keep `ward_matric_aggs` from `postsec`?

[master](https://github.com/almartin82/njschooldata/blob/4d873fa2464b62df7ba38eb241d61453ac8385e0/R/geo.R) vs [postsec](https://github.com/almartin82/njschooldata/blob/4027f2f0c09b01dac057c343b100cc21e18426c4/R/geo.R) were the two branches

**Review by almartin82 — APPROVED** [review]

**Comment by chiouey** [comment]
> Looks good!
> 
> There was a merge conflict on `geo.R` - I took a look at both files and resolved in [3179c34](https://github.com/almartin82/njschooldata/commit/3179c3411b6f9d0b531f7de8eb71bde579ff15d5) - @chiouey can you take a look at that and confirm? `ward_gcount_aggs` goes away and we keep `ward_matric_aggs` from `postsec`?
> 
> [master](https://github.com/almartin82/njschooldata/blob/4d873fa2464b62df7ba38eb241d61453ac8385e0/R/geo.R) vs [postsec](https://github.com/almartin82/njschooldata/blob/4027f2f0c09b01dac057c343b100cc21e18426c4/R/geo.R) were the two branches

Huh -- not totally sure what happened there. `ward_gcount_aggs` and `ward_grate_aggs` definitely should still be around. Maybe I didn't use the right branch as base? How can I fix?

@almartin82

**Comment by almartin82** [comment]
ok!  I will find those functions and restore

**Comment by almartin82** [comment]
@chiouey restored those

**Pull request merged**